tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+###
+### Main entrypoint
+###
+
+- name: Check if Helm binary exists
+  shell: command -v helm >/dev/null 2>&1
+  register: helm_exist
+  ignore_errors: yes
+
+- name: Fail if Helm command does NOT exist
+  fail:
+    msg: "Helm command doesn NOT exist. Please install Helm v{{ helm_version }}"
+  when: helm_exist.rc != 0
+
+- name: Get Helm verson
+  command: "helm version --client --template '{{ '{%' }} raw %}{{ '{{' }} .Client.SemVer }}{{ '{%' }} endraw %}'"
```

Comment:
to improve the maintainability of this, I suggest going with `!unsafe`. see [this](https://docs.ansible.com/ansible/latest/user_guide/playbooks_advanced_syntax.html#unsafe-or-raw-strings) for more details.

Reply:
After reading, I don't get why we would need `!unsafe`? 

(...)preventing unsafe character subsitition and information disclosure(...)

Reply:
Because, Ansible's a very big fan (NOT) of golang templates. It sees the moustaches and thinks it's Jinja.

What you basically do with:
`'{{ '{%' }} raw %}{{ '{{' }} .Client.SemVer }}{{ '{%' }} endraw %}'`

can be done like:

`!unsafe '{{ .Client.SemVer }}'`

I think it's cleaner, thus more maintainable. Besides, Jinja's `{%raw%}` is not _safe_ either, if that's your concern.

Reply:
When I change it to:

```yaml
- name: Get Helm version
  command: !unsafe "helm version --client --template '{{ .Client.SemVer }}'"
  register: _helm_local_version
  changed_when: False
  ignore_errors: True
```

I get:

```
TASK [rolename : Get Helm version] **********************************************************************************************************************
Friday 22 March 2019  12:59:40 +0000 (0:00:00.014)       0:00:00.533 ********** 
fatal: [localhost]: FAILED! => {"msg": "template error while templating string: unexpected '.'. String: helm version --client --template '{{ .Client.SemVer }}'"}
...ignoring
```

Reply:
That's because `!unsafe` is to be used for variable values. For example:
```
  vars:
    helm_version: !unsafe helm version --client --template '{{ .Client.SemVer }}'
  tasks:
    - name: Get Helm version
      command: "{{ helm_version }}"
      register: _helm_local_version
      changed_when: False
      ignore_errors: True
```